tests/components/vizio/test_media_player.py

```diff
@@ -9,23 +9,14 @@
 import pytest
 from pytest import raises
 from pyvizio.api.apps import AppConfig
-from pyvizio.const import (
-    APPS,
-    DEVICE_CLASS_SPEAKER as VIZIO_DEVICE_CLASS_SPEAKER,
-    DEVICE_CLASS_TV as VIZIO_DEVICE_CLASS_TV,
-    INPUT_APPS,
-    MAX_VOLUME,
-    UNKNOWN_APP,
-)
+from pyvizio.const import APPS, INPUT_APPS, MAX_VOLUME, UNKNOWN_APP
```

Comment:
I don't think VIZIO_DEVICE_CLASS_SPEAKER and VIZIO_DEVICE_CLASS_TV should be removed.
They come from pyvizio and are not HA constants.

Reply:
Good spot, thanks, should be sorted now.